e2e/assets/vulnerable_rust_deps/src/hello/Cargo.toml

```diff
@@ -8,6 +8,6 @@ crate-type = ["cdylib"]
 
 [dependencies]
 candid = "0.7.4"
-ic-cdk = "0.4"
-ic-cdk-macros = "0.4"
-time = "0.1.43"
\ No newline at end of file
+ic-cdk = "0.7"
+ic-cdk-macros = "0.7"
```

Comment:
This version doesn't actually exist.

Reply:
https://crates.io/crates/ic-cdk
https://crates.io/crates/ic-cdk-macros

downgraded `-macros` to `0.6`